cli/src/commands/prev.rs

```diff
@@ -21,10 +21,15 @@ use crate::command_error::{user_error, CommandError};
 use crate::commands::next::choose_commit;
 use crate::ui::Ui;
 
-/// Move the working-copy commit to the parent revision
+/// Move the working-copy commit relative to the parent revision
```

Comment:
If we use relative here, we need to adjust a bunch of other documentation.

Reply:
Good point. Let's finalize this help page and then update everything else to match.

Reply:
Is it a `new` commit or a reparenting of existing commit or something else? 
(I think "Move" is confusing.)

Reply:
> Is it a `new` commit or a reparenting of existing commit or something else?

It is a `new` commit which is pretends to be a rebased one.

Reply:
You can continue with `next` now, as `prev` has improved.